<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editorwindow</class>
 <widget class="QMainWindow" name="editorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>437</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>770</y>
      <width>437</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(43, 43, 43);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background: rgb(43, 43, 43);
color: white;
border: 1px solid transparent;
margin: auto;</string>
         </property>
         <property name="text">
          <string>Brilho</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="saturacao">
         <property name="styleSheet">
          <string notr="true">background: rgb(43, 43, 43);
color: white;
border: 1px solid transparent;
margin: auto;</string>
         </property>
         <property name="text">
          <string>Saturação</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="girar">
         <property name="styleSheet">
          <string notr="true">background: rgb(43, 43, 43);
color: rgb(43, 43, 43);
border: 1px solid transparent;
margin: auto;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="req.qrc">
           <normaloff>:/refresh-arrow.png</normaloff>:/refresh-arrow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QWidget" name="widget_3" native="true">
         <widget class="QSlider" name="sliderSaturacao">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>2</y>
            <width>221</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="accessibleName">
           <string>slider</string>
          </property>
          <property name="accessibleDescription">
           <string>slider</string>
          </property>
          <property name="minimum">
           <number>-50</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="tracking">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>2</y>
            <width>221</width>
            <height>22</height>
           </rect>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>43</red>
                <green>43</green>
                <blue>43</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="accessibleName">
           <string>slider</string>
          </property>
          <property name="accessibleDescription">
           <string>slider</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>-50</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="tracking">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="girarLabel">
         <property name="styleSheet">
          <string notr="true">background: rgb(43, 43, 43);
color: white;
border: 1px solid transparent;
margin: auto;
</string>
         </property>
         <property name="text">
          <string>    Girar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>770</y>
      <width>437</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(43, 43, 43);
color: white;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">background: rgb(43, 43, 43);
</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>518</width>
          <height>62</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="peb">
              <property name="styleSheet">
               <string notr="true">border-radius: 5px;
border: solid 1px transparent;
background: rgb(57, 77, 99);
color: white;
padding-left: 20px;
padding-right: 20px;
padding-top: 10px;
padding-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>Preto e Branco</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="blur">
              <property name="styleSheet">
               <string notr="true">border-radius: 5px;
border: solid 1px transparent;
background: rgb(57, 77, 99);
color: white;
padding-left: 20px;
padding-right: 20px;
padding-top: 10px;
padding-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>Desfoque</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="inverter">
              <property name="styleSheet">
               <string notr="true">border-radius: 5px;
border: solid 1px transparent;
background: rgb(57, 77, 99);
color: white;
padding-left: 20px;
padding-right: 20px;
padding-top: 10px;
padding-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>Inverter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sepia">
              <property name="styleSheet">
               <string notr="true">border-radius: 5px;
border: solid 1px transparent;
background: rgb(57, 77, 99);
color: white;
padding-left: 20px;
padding-right: 20px;
padding-top: 10px;
padding-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>Sépia</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="special">
              <property name="styleSheet">
               <string notr="true">border-radius: 5px;
border: solid 1px transparent;
background: rgb(57, 77, 99);
color: white;
padding-left: 20px;
padding-right: 20px;
padding-top: 10px;
padding-bottom: 10px;</string>
              </property>
              <property name="text">
               <string>FX01</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>437</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(43, 43, 43);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="voltar">
       <property name="styleSheet">
        <string notr="true">background: rgb(43, 43, 43);
border: 1px solid transparent;
margin: auto;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="req.qrc">
         <normaloff>:/undo.png</normaloff>:/undo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="default_2">
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
border: solid 1px transparent;
background: rgb(40, 146, 252);
color: white;
padding: 2px;
</string>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>43</red>
             <green>43</green>
             <blue>43</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: auto;
background: rgb(43, 43, 43);
border: 1px solid transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="req.qrc">
         <normaloff>:/diskette.png</normaloff>:/diskette.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>740</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(43, 43, 43);
border: 1px solid transparent ;
border-bottom:  2px solid rgb(28, 28, 28);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="ferramentas">
       <property name="styleSheet">
        <string notr="true">
background: rgb(43, 43, 43);
color: white;
border: 1px solid transparent;
padding: 2px;
</string>
       </property>
       <property name="text">
        <string>Ferramentas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filtros">
       <property name="styleSheet">
        <string notr="true">
background: rgb(43, 43, 43);
color: white;
border: 1px solid transparent;

padding: 2px;

</string>
       </property>
       <property name="text">
        <string>Filtros</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>widget_5</zorder>
   <zorder>graphicsView</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="req.qrc"/>
 </resources>
 <connections/>
</ui>
